build/Dockerfile.nginxplus

```diff
@@ -38,4 +38,4 @@ USER 101:1001
 
 LABEL org.nginx.ngf.image.build.agent="${BUILD_AGENT}"
 
-CMD ["nginx", "-g", "daemon off;"]
+CMD ["sh", "-c", "rm -rf /var/run/nginx/* /var/lib/nginx/*.sock && nginx -g 'daemon off;'"]
```

Comment:
if the problem is only with unix sockets, shall we only remove `*.sock`, not `/*` ?

also, let's move the socket `/var/run/nginx/nginx-status.sock` to `/var/lib/nginx/nginx-status.sock`? for consistency and we will have one less folder to clean up.

Reply:
@pleshakov Does it not make more sense to have all socket files in `/var/run`? [reference](https://www.pathname.com/fhs/pub/fhs-2.3.html#VARRUNRUNTIMEVARIABLEDATA)

Reply:
Either way, we also need to update a couple docs (and maybe remove an unnecessary volume mount?)

Reply:
@sjberman I don't see any /var/lib directory here https://hg.nginx.org/pkg-oss/file/tip/alpine/Makefile except for Plus mgmt module, so I think it should be ok to remove it.
it is used to be used before for temp files and stuff. for example https://trac.nginx.org/nginx/ticket/750